code/game/objects/items/storage/bags.dm

```diff
@@ -39,7 +39,6 @@
 	inhand_icon_state = "trashbag"
 	lefthand_file = 'icons/mob/inhands/equipment/custodial_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/custodial_righthand.dmi'
-	slot_flags = null
```

Comment:
is CI failing because this line was removed? that's odd. i could be wrong though but that seems to be the only thing that could be causing the missing icon state? unsure.

Reply:
I think it's cause they don't have a worn icon state, which do be depressing

Reply:
handled